subworkflows/local/asset_to_channel/main.nf

```diff
@@ -143,7 +143,8 @@ workflow ASSET_TO_CHANNEL {
     vcf_branch = asset.branch { meta, _fasta ->
         file: meta.vcf
         // If we already have the vcf_tbi, then we don't need to index the vcf
-        def meta_extra = [run_tabix: meta.vcf_tbi ? false : true]
+        def meta_extra = [run_tabix: meta.vcf_tbi || meta.vcf.endsWith('.vcf') ? false : true]
```

Comment:
Appears if `vcf_tbi` is present but `vcf` is not compressed, compression will be skipped.  Is that the intent?

Reply:
I assume that if we have a tbi, it means we have a gz